src/platforms/generic_unix/lib/platform_nifs.c

```diff
@@ -53,120 +53,6 @@
     return term_invalid_term();
 
 #if defined ATOMVM_HAS_OPENSSL
```

Comment:
Are we basically ditching openssl support except for random numbers?

Reply:
The idea is to completely drop dependence on OpenSSL indeed, starting with the easiest (crypto). It's only used on generic_unix and we're probably not going to maintain SSL over OpenSSL.

Random is more work and should be discussed separately. Do we want to implement crypto:rand_bytes and crypto:strong_rand_bytes? Do we want to implement random module? I understand we have atomvm:random because of ESP32's API and the generic_unix and emscripten versions are backported...

Reply:
> The idea is to completely drop dependence on OpenSSL indeed, starting with the easiest (crypto). It's only used on generic_unix and we're probably not going to maintain SSL over OpenSSL.

I can agree on this, we can even easily static link it when building Linux artifacts removing all those openssl version dependent artifacts.

> Do we want to implement crypto:rand_bytes

Deprecated and removed, so I'd say no.

> and crypto:strong_rand_bytes?

Yes.